crates/nu-command/src/network/fetch.rs

```diff
@@ -361,6 +361,15 @@ fn response_to_buffer(
     engine_state: &EngineState,
     span: Span,
 ) -> nu_protocol::PipelineData {
+    // Try to get the size of the file to be downloaded.
+    // This is helpful to show the progress of the stream.
+    let buffer_size = match &response.headers().get("content-length") {
+        Some(content_length) => {
+            let tmp_header = content_length.clone(); // binding
+            Some(tmp_header.to_str().unwrap().parse::<usize>().unwrap())
```

Comment:
The `unwrap()` calls here need to be removed - we don't want to crash Nushell just because we got a bad `content-length` header.